airmencoders/Logging-iOS
# PBS-1495 Delete empty Issue/Aircrew rows

[PBS-1495](https://jira.il2.dso.mil/browse/PBS-1495)

**Discussion** 🗣
This PR adds functionality to remove empty rows (rows with no fields filled in) from the IssuesListView and the AircrewListView. The functions are called when navigating away from the views containing the lists. 

**Important Notes** ✍️
- The functions iterate backwards through the lists, as removing elements in a forwards order results in an index out of range error.
- One thing I noticed when testing: If you add an empty row to the Aircrew list and then navigate back to the previous view, you can see the empty row appear, and then quickly disappear from the static Aircrew list. Would be interested to hear thoughts on if we should address that or not. 

**Testing** ✅
- Add empty rows, ensure they disappear when navigating away.
- Add empty rows, and fill in only one field. Ensure the rows persist after navigating away and back. Repeat for all fields in the row. 
- Do this with both the Issues list and the Aircrew list.
